theme_graphene/views/customizations.xml

```diff
@@ -45,29 +51,27 @@
 
 <!-- ======== TEXT-IMAGE ======== -->
 <template id="s_text_image" inherit_id="website.s_text_image" name="Graphene s_text_image">
-    <xpath expr="//section" position="attributes">
-        <attribute name="class" add="pb96 pt0" remove="pb80 pt80" separator=" "/>
-    </xpath>
     <xpath expr="//h2" position="replace">
-        <h3>Unique experiences to drive engagement</h3>
+        <h2>Unique experiences to drive engagement</h2>
         <div class="s_hr pt16 pb0" data-snippet="s_hr" data-name="Separator">
             <hr class="w-100 mx-auto" style="border-top-color: rgba(0, 0, 0, 0) !important;"/>
         </div>
     </xpath>
+    <xpath expr="//img" position="attributes">
+        <attribute name="src">/web_editor/image_shape/website.s_image_text_device_perspective/web_editor/devices/iphone_3d_portrait_02.svg</attribute>
+        <attribute name="data-shape">web_editor/devices/iphone_3d_portrait_02</attribute>
+    </xpath>
```

Comment:
The editor doesn't work properly. You can't modify the shape and a few other options. Maybe you just need to add a few more attributes for it to work :thinking: (cf. something similar is done in `theme_odoo_experts/views/snippets/s_image_text.xml`)

| wrong | right |
| --- | --- |
| ![image](https://github.com/user-attachments/assets/18126e69-90b9-4f20-9d48-89da1cadb381) | ![image](https://github.com/user-attachments/assets/80fd24ca-0ba4-42e1-826f-0a29c5f8f63a) |

Reply:
Good catch! 

It took me some time to figure out, we need a `s_image_text_device` record for it to work. I guess there is some black magic somewhere in the code which shows the properties of this record, while visually showing the `_perspective` image version of it in the device shape for the first load. 🤯 

It should be fine now 😄

Reply:
Oh well done, I had no idea :exploding_head: It's good to know. I found the JS responsible for it in web_editor:

```js
let match = img.src.match(/\/web_editor\/image_shape\/(\w+\.\w+)/);
if (img.dataset.shape && match) {
    match = match[1];
    if (match.endsWith("_perspective")) {
        // As an image might already have been modified with a
        // perspective for some customized snippets in themes. We need
        // to find the original image to set the 'data-original-src'
        // attribute.
        match = match.slice(0, -12);
    }
    return this._loadImageInfo(`/web/image/${encodeURIComponent(match)}`);
}
```